source/ProgArgs.h

```diff
@@ -146,6 +146,7 @@ namespace bpt = boost::property_tree;
 #define ARG_S3BUCKETACLPUT_LONG		"s3baclput"
 #define ARG_S3ENDPOINTS_LONG		"s3endpoints"
 #define ARG_S3FASTGET_LONG			"s3fastget"
+#define ARG_S3IGNOREERRORS_LONG	"s3ignoreerrors"
```

Comment:
alignment of the word "s3ignoreerrors" not correct. 
tab width is 4. i actually switched from tabs to 4 spaces recently for new patches. but not a showstopper if you keep tabs for this patch (because i know i should really write a code style guide for these things. some day... :-) )

Reply:
yeah, sorry. some of the parts of the file was still tabs and some other spaces. would love the mega patch from you to standardize at some point <3

Reply:
i'm still thinking about that mega patch, but probably won't do it, because this change to all code lines would make "git blame" harder and thus i wouldn't even know who to yell at if there ever is a bug in the code ;-)